capsules/src/max17205.rs

```diff
@@ -13,12 +13,15 @@
 //! -----
 //!
 //! ```rust
-//! let max17205_i2c = static_init!(
+//! let max17205_i2c0 = static_init!(
```

Comment:
So, this chip responds to _two_ different I2C addresses simultaneously?

Reply:
Yes it does respond to two different i2c addresses depending on which address you are writing to 0x000-0x0FF is address 0x36, and 0x100-1FF is address 0x0B. We could name them i2c_lower, and i2c_upper or something like that to denote the address spaces they access and put a better commend in the code explaining why it needs two I2C.

Reply:
How weird. I don't know that I've ever seen an IC that responds to different I2C addresses based on which command you're writing/reading. Do you think it somehow allowed them to reuse a chunk of silicon?

Like you said, a good comment about the chip would be wonderful since this is odd behavior.